specificatie/verblijfplaats.yaml

```diff
@@ -103,8 +108,8 @@ components:
           $ref: '#/components/schemas/Postcode'
         woonplaats:
           $ref: '#/components/schemas/Woonplaats'
-        korteNaam:
-          $ref: '#/components/schemas/KorteNaam'
+        korteStraatnaam:
```

Comment:
@MelvLee @KayodeBakker gooien we voor jullie de boel erg in de war wanneer we de velden hier eindelijk eens in een logische volgorde zetten? Dus korteStraatnaam direct voor of direct achter officieleStraatnaam?

Reply:
Is het maar een volgorde. Ik merk er niks van.